README.md

```diff
@@ -17,10 +17,18 @@ chktex -q -l .chktexrc src/main.tex
 
 #### TeXtidote
 
+Report:
+
 ```shell
 textidote src/main.tex
 ```
 
+Process analysis:
+
+```shell
+textidote src/process-analysis/process-analysis.tex
```

Comment:
Why is this necessary?

Reply:
Using the command "textidote src/main.tex" does not analyze the documents related to the process analysis, why i found it nice to have the other command at hand in the READ.ME